server/controllers/profile.controller.js

```diff
@@ -71,6 +71,68 @@ class ProfileController {
     delete dataValues.password;
     return Response(res, 200, 'Profile updated successfully.', [dataValues]);
   }
+
+  /**
+     * @description follows another user
+     * @param {object} req
+     * @param {object} res
+     * @returns {object} user that has been followed
+     */
+  static async followUser(req, res) {
+    const { username } = req.params;
+    const { id } = req.user;
+    const requestingFollowership = await User.findByPk(id);
+    const personToBeFollowed = await User.findOne({ where: { username } });
+    const successMessage = 'Successfully followed user';
+
+    if (!personToBeFollowed) {
+      return Response(res, 404, 'You cannot follow a user that does not exist');
```

Comment:
Keep the response simple and straight to the point. `User does not exist` could suffice